### PR TITLE
Adding testbook package

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- ~~[ ] If static libraries are linked in, the license of the static library is packaged.~~
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by fcollonval** [comment]
@MSeal @rohitsanj Could you confirm you agree to be maintainers?

File: recipes/testbook/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{% set name = "testbook" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3aef6aa44de9accf62839e8c9f471fd558add64eca64056729dea08824589756
```
Comment (dbast) [inline recipes/testbook/meta.yaml:10]: Seems to be not a valid URL... this also does not list any tar.gz sdist file: https://pypi.org/project/testbook/#files

This should work instead:

```suggestion
  url: https://github.com/nteract/{{ name }}/archive/{{ version }}.tar.gz
  sha256: f35f66f3c074cb05e3d51000a5aaf7355a48826e29365f15b9681c0e998f35da
```

File: recipes/testbook/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - nbformat >=5.0.4
+    - nbclient >=0.4.0
+
+test:
+  imports:
+    - testbook
```
Comment (dbast) [inline recipes/testbook/meta.yaml:28]: Please extend to

```suggestion
  requires:
    - pip
  imports:
    - testbook
  commands:
    - pip check
```

**Comment by fcollonval** [comment]
Thanks @dbast for the review and corrections.

I got the authors agreement to list them as maintainer: https://github.com/nteract/testbook/issues/60#issuecomment-722295507

So if the actions pass, it is good to go.

**Pull request merged**